app/controllers/claims_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ClaimsController < ApplicationController
+  def new
+    contribution = Listing.find(params[:contribution_id])
+    if contribution.person&.email.blank?
+      flash.now[:alert] = "We are sorry, this contributor hasn't provided
+                         an email address yet and can't communicate this way".squish
+    end
+    render locals: { contribution: contribution }
```

Comment:
Interesting use of an explicit `render` with `locals` instead of relying on setting instance variables.

Its a little more typing but i do like the explicit definition of what gets passed to the view, and have always disliked the magic ivars in templates!

Is this a norm y'all at PHR follow? I'm definitely ok with it but would like to hear your thoughts @h-m-m, @maebeale and @svileshina. We don't use this form in any of our existing controllers. What do you think? (Just to be clear what i'm talking about, in our code base we would drop line 8 above and change line 3 to be `@contribution = ...`.)

Reply:
Thanks for @indiebrain, he recently introduced this concept to me. I think it's a good way to inject dependencies into views and also comes in handy during refactoring too. I'm definitely open to reverting it to use ivars.

Reply:
I like it as well and very open to adopting it. Lets keep it as is in this PR and that'll give our team a chance to discuss the convention. 🤠